### DISCUSSION TITLE
Bay multipolygons end up being inside out

### DISCUSSION THREAD

**Discussion by adjourn** [onthegomap/planetiler#1069] [Q&A]
Warning! Contains high art.. 🙃 

Original OSM data looks something like this. It's a multipolygon that could contain multiple "holes" or islands to be more specific.

![osm](https://github.com/user-attachments/assets/90acd38a-8c16-4ad8-b44f-cb43ac93b592)

However the planetiler output looks like this (for all the big bays that contain islands):

![output](https://github.com/user-attachments/assets/f3bcb6ff-c547-43a5-9478-2535443e5d33)

If bay contains multiple islands, all island areas ("holes", "cutouts", whatever you want to call them) end up being bay polygons.

Not doing anything fancy here:
```java
if (sourceFeature.canBePolygon() && sourceFeature.hasTag("natural", "bay")) {
   var feature = features.polygon("testing")
        .setAttr("type", "bay")
        .setZoomRange(0, 14)
        .setMinPixelSize(1)
        .setBufferPixels(8);
}
```

Any advice is highly appreciated!

Not sure if it's relevant but I'm using `--area` to generate specific countries. I have no idea how data is "sliced". Could it be that geofabrik cuts off the multipolygon? Or does it always contain the full geometry even if it's overhangs the selected "area"?

**Comment by msbarry** [comment]
Hello! That code looks fine, there could be any number of issues going on here.  Could you share a minimal test project (input file, code, and how you are building it) to reproduce this?

**Reply by adjourn** [reply]
Back to mapping! Had lots of stuff going on in past.. Many months.
I will provide a minimal test project if I return to this issue!
